instat/dlgDotPlot.vb

```diff
@@ -44,10 +43,11 @@ Public Class dlgDotPlot
         autoTranslate(Me)
         TestOkEnabled()
     End Sub
-    Private Sub SetRCodeForContorls(bReset As Boolean)
-        ucrDotPlotSelector.SetRCode(clsRggplotFunction, bReset)
+
+    Public Sub SetRCodeForContorls(bReset As Boolean)

```

Comment:
Since this dialog will have a sub dialog that calls this, I assume it should be a public sub?